src/core/RestValue.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {IdentifiedDataSerializable, IdentifiedDataSerializableFactory} from '../serialization/Serializable';
+import {DataInput, DataOutput} from '../serialization/Data';
+
+export const REST_VALUE_FACTORY_ID = -37;
+export const REST_VALUE_CLASS_ID = 1;
+
+export class RestValue implements IdentifiedDataSerializable {
+    value: number[];
```

Comment:
For both `value` and `contentType` fields, it's better to use native type for binary data (`Buffer`) instead of an array of numbers. But in this case we could also use strings instead of numbers, as they come from HTTP. It's not the way we do it in Java client, but over there it's used also in REST API, so that should be the reason why it operates binary data instead of strings. WDYT?

BTW we have a PRD to improve the client in this way:
https://hazelcast.atlassian.net/wiki/spaces/PM/pages/1749254626/Node.js+Client+Support+native+binary+data+type

@burakcelebi maybe it makes sense to include the above PRD into the 4.0 plan, as it assumes backwards incompatible changes. WDYT?

Reply:
Changed field types to string